### ISSUE TITLE
App is stuck at splashscreen and crashing with SIGSEGV, code 1 after user has done an update from playstore.

### ISSUE THREAD

**Issue by arjun159** [xamarin/xamarin-android#5446]
fter the user had updated app in playstore to a later version in playstore, xamarin forms had a crash of SIGSEV in splash screen. The issue has been happening only in production

Xamarin forms version -- 4.7.0.1351
Xamarin Android SDK version -- 11.1.0.17

Most of the devices affected are Huawei, Samsung with different versions of android.

The issue has been happening in the release mode and only after an update installed.

Steps to reproduce : 

1. App is already present in playstore
2. Organisation has given a new update for the app.
3. when user has updated the app app is stuck in spalshscreen and has a crash of SIGSEGV, code 1 

Logs : 

*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
pid: 0, tid: 0 >>> <<<

backtrace:
  #00  pc 000000000001e8a0  /system/lib64/libc.so (strlen+16)
  #00  pc 000000000006f6e8  /system/lib64/libc.so (strdup+20)
  #00  pc 0000000000013e44  /data/app/be.scarlet.mobile-qBgysgJq15aUbi5hn9oRLg==/lib/arm64/libmonodroid.so (xamarin::android::internal::MonodroidRuntime::get_java_class_name_for_TypeManager(_jclass*)+92)
  #00  pc 000000000000f028  <anonymous>

**Comment by edsulaiman** [comment]
Give me the play store link of your application, I'll test it. 

from what i know, SIGSEV means a signal for memory access violation, trying to read or write from/to a memory area that your process does not have access to

**Comment by jpobst** [comment]
With support for Classic Xamarin.Android ended [May 1st, 2024](https://dotnet.microsoft.com/en-us/platform/support/policy/xamarin), this issue is likely no longer relevant.

If this still persists in .NET 8+, please open a new issue with updated information based on `net8.0-android` or greater.  Please include a link to this issue for context.
